api/reqs/models.py

```diff
@@ -108,30 +108,32 @@ def __str__(self):
 class Policy(models.Model):
     class Meta:
         verbose_name_plural = ugettext_lazy('Policies')
-        ordering = ['policy_number']
+        ordering = ['policy_number', 'pk']
 
-    policy_number = models.IntegerField(unique=True)
     title = models.CharField(max_length=1024)
-    uri = models.CharField(max_length=256)
     omb_policy_id = models.CharField(max_length=64, blank=True)
-    policy_type = models.CharField(
-        max_length=32, choices=[(e.name, e.value) for e in PolicyTypes],
-        blank=True
-    )
     slug = models.SlugField(max_length=title.max_length)
     issuance = models.DateField()
     sunset = models.DateField(blank=True, null=True)
-    policy_status = models.CharField(max_length=256, blank=True)
-    document_source = models.FileField(blank=True)
     public = models.BooleanField(default=True)
-    issuing_body = models.CharField(max_length=512)
-    managing_offices = models.ManyToManyField(
-        Office, blank=True, related_name='policies')
```

Comment:
I have this feeling that a bunch of the above are going to come back, but I can see the argument for removing them now.

Reply:
Agreed! I suspect they might also come back on a different form.